pkg/api/v1beta3/types.go

```diff
@@ -653,59 +653,59 @@ type BoundPod struct {
 // the pod list is guaranteed to only change when the list of bound pods changes.
 type BoundPods struct {
 	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty"`
+	ObjectMeta `json:"metadata,omitempty" description:"standard object metadata; see https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-conventions.md#metadata"`
 
 	// Host is the name of a node that these pods were bound to.
-	Host string `json:"host"`
+	Host string `json:"host" description:"name of a node that these pods were bound to"`
 
 	// Items is the list of all pods bound to a given host.
-	Items []BoundPod `json:"items"`
+	Items []BoundPod `json:"items" description:"list of all pods bound to a given host"`
 }
 
 // ReplicationControllerSpec is the specification of a replication controller.
 type ReplicationControllerSpec struct {
 	// Replicas is the number of desired replicas.
-	Replicas int `json:"replicas"`
+	Replicas int `json:"replicas" description:"number of replicas desired"`
 
 	// Selector is a label query over pods that should match the Replicas count.
-	Selector map[string]string `json:"selector,omitempty"`
+	Selector map[string]string `json:"selector,omitempty" description:"label keys and values that must match in order to be controlled by this replication controller"`
 
 	// TemplateRef is a reference to an object that describes the pod that will be created if
 	// insufficient replicas are detected.
-	TemplateRef *ObjectReference `json:"templateRef,omitempty"`
+	TemplateRef *ObjectReference `json:"templateRef,omitempty" description:"templateRef is a reference to an object that describes the pod that will be created if insufficient replicas are detected"`
```

Comment:
In general, naming the field in the description is not necessary/desirable. Just start "reference to...".